server/data/visitSchedulerApiTypes.ts

```diff
@@ -1,10 +1,15 @@
 import { components, operations } from '../@types/visit-scheduler-api'
 
-export type SessionLocationGroup = components['schemas']['SessionLocationGroupDto']
```

Comment:
I've renamed this to simply `LocationGroup` which I think will be a bit clearer in our code? Happy to be persuaded to keep the full name though...🤷‍♂️